.gitattributes

```diff
@@ -0,0 +1,5 @@
+# line endings
+* text=auto eol=lf
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
+*.{vcxproj,vcxproj.filters} text eol=crlf
```

Comment:
Where do these come from? There aren't any cmd/bat files in the repo, are there?

Reply:
That's the template I use by default everywhere without the need for manually checking the content of the repos.